<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppearanceWidget</class>
 <widget class="QWidget" name="AppearanceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>343</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Apearance</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_active">
      <attribute name="title">
       <string>Active desktop</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea_active">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_active">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>448</width>
            <height>289</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="layout_fontActive">
             <item>
              <widget class="QLabel" name="label_fontActive">
               <property name="text">
                <string>Font</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFontComboBox" name="fontComboBox_fontActive"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="layout_fontSizeActive">
             <item>
              <widget class="QLabel" name="label_fontSizeActive">
               <property name="text">
                <string>Font size</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_fontSizeActive">
               <property name="minimum">
                <number>7</number>
               </property>
               <property name="maximum">
                <number>48</number>
               </property>
               <property name="value">
                <number>12</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="layout_fontColorActive">
             <item>
              <widget class="QLabel" name="label_fontColorActive">
               <property name="text">
                <string>Font color</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="KColorCombo" name="kcolorcombo_fontColorActive"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="layout_fontWeightActive">
             <item>
              <widget class="QLabel" name="label_fontWeightActive">
               <property name="text">
                <string>Font weight</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_fontWeightActive">
               <property name="minimum">
                <number>100</number>
               </property>
               <property name="maximum">
                <number>900</number>
               </property>
               <property name="singleStep">
                <number>100</number>
               </property>
               <property name="value">
                <number>400</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="layout_fontStyleActive">
             <item>
              <widget class="QLabel" name="label_fontStyleActive">
               <property name="text">
                <string>Font style</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_fontStyleActive">
               <item>
                <property name="text">
                 <string notr="true">normal</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string notr="true">italic</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_active">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>119</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_inactive">
      <attribute name="title">
       <string>Inactive desktop</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea_inactive">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_inactive">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>448</width>
            <height>289</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="layout_fontInactive">
             <item>
              <widget class="QLabel" name="label_fontInactive">
               <property name="text">
                <string>Font</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFontComboBox" name="fontComboBox_fontInactive"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="layout_fontSizeInactive">
             <item>
              <widget class="QLabel" name="label_fontSizeInactive">
               <property name="text">
                <string>Font size</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_fontSizeInactive">
               <property name="minimum">
                <number>7</number>
               </property>
               <property name="maximum">
                <number>48</number>
               </property>
               <property name="value">
                <number>12</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="layout_fontColorInactive">
             <item>
              <widget class="QLabel" name="label_fontColorInactive">
               <property name="text">
                <string>Font color</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="KColorCombo" name="kcolorcombo_fontColorInactive"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="layout_fontWeightInactive">
             <item>
              <widget class="QLabel" name="label_fontWeightInactive">
               <property name="text">
                <string>Font weight</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_fontWeightInactive">
               <property name="minimum">
                <number>100</number>
               </property>
               <property name="maximum">
                <number>900</number>
               </property>
               <property name="singleStep">
                <number>100</number>
               </property>
               <property name="value">
                <number>400</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="layout_fontStyleInactive">
             <item>
              <widget class="QLabel" name="label_fontStyleInactive">
               <property name="text">
                <string>Font style</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_fontStyleInactive">
               <item>
                <property name="text">
                 <string notr="true">normal</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string notr="true">italic</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_inactive">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>119</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorCombo</class>
   <extends>QComboBox</extends>
   <header>kcolorcombo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
